.github/workflows/cache-issues.yaml

```diff
@@ -3,15 +3,22 @@ on:
   schedule:
     - cron: '0 */6 * * *'
   workflow_dispatch:
+  # TODO: Add for test, remove before merge
+  push:
+  pull_request:
```

Comment:
Sounds like undesired part was merged.

Reply:
yes, I reverted it out

since for testing those github action, I open the PR from the main repo.
I did a mistake and push the changes to my fork, and merged.


anyhow I've pull back the unrelated commit from master